### ISSUE TITLE
Minimap search decorations are hard to see

### ISSUE THREAD

**Issue by misolori** [microsoft/vscode#76122]
Testing #76030

![image](https://user-images.githubusercontent.com/35271042/60121224-739a5380-9737-11e9-8e5c-1e577b07d906.png)

I get that you're using the same color as the highlighted text in the editor, but this is hard to see even if you're looking for it. I wonder if it's worth exploring to try a different color in our default themes?

**Comment by usernamehw** [comment]
Minimap appears to be too small for decorations. Maybe you can consider making an option to scale it for the next iteration? #21773

**Comment by usernamehw** [comment]
Highlighting the entire line could also help https://github.com/microsoft/vscode/issues/21404#issuecomment-294275446

![](https://cloud.githubusercontent.com/assets/17151327/25061366/8a65c63a-2182-11e7-9f26-09f518324ac9.png)

**Comment by stiegi** [comment]
No, it should be just like in Sublime or Atom: a bright, white border around the result. It should also be there after doubleclicking something, and NOT the complete line, because this feature is supposed to help finding the horizontal position, too.

**Comment by usernamehw** [comment]
Line highlighting could be half/more than half transparent so that it would be easy to fine the line and the precise horizontal location (like diff, but background should be more transparent):

![Screenshot (18)](https://user-images.githubusercontent.com/9638156/60755240-96f5b600-9ff5-11e9-8651-47923f842562.png)

**Comment by RMacfarlane** [comment]
I like the idea of highlighting the whole line. Here’s what that looks like in a prototype:
![Screen Shot 2019-07-10 at 2 30 15 PM](https://user-images.githubusercontent.com/3672607/61006542-6c6e6a80-a31f-11e9-88c2-d8d92a62a9f3.png)

I defined a new theme color, `minimap.findMatchHighlight`, which by default is the same as the overview ruler but opaque. As @usernamehw suggested, I added a line highlight that’s half transparent.

I’ll check this in for now, still happy to take more feedback and experiment more over the iteration.
